_maps/RandomRuins/IceRuins/icemoon_surface_bughabitat.dmm

```diff
@@ -297,7 +297,7 @@
 	dir = 4
 	},
 /obj/structure/barricade/wooden/crude,
-/turf/closed/mineral/random/snow,
+/turf/open/space/basic,
```

Comment:
huh? there shouldnt be space on an ice planet ruin

Reply:
Fixing